extensions/http-status-codes/src/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { ActionPanel, Action, Icon, List } from "@raycast/api";
+import React from "react";
+import http from "http";
+
+type Code = {
+  code: string;
+  description?: string;
+};
+
+export default function Command() {
+  const codeGroups = Object.entries(http.STATUS_CODES).reduce(
+    (groups: { [firstDigit: string]: Code[] }, [code, description]) => {
+      const group = groups[code[0]] || [];
+      group.push({ code, description });
+      groups[code[0]] = group;
+
+      return groups;
+    },
+    {}
+  );
+
+  return (
+    <List isLoading={false} searchBarPlaceholder="Filter by code or description...">
+      {Object.entries(codeGroups).map(([firstDigit, codes]) => (
+        <List.Section key={firstDigit} title={`${firstDigit}xx`} subtitle={getCodeGroupDescription(firstDigit)}>
+          {codes.map((code) => (
+            <List.Item
+              key={code.code}
+              title={code.code}
+              subtitle={code.description}
+              keywords={[code.description || ""]} // make subtitle searchable
+              icon={{
+                source: Icon.Dot,
+                tintColor: statusCodeToColor(code.code),
+              }}
+              actions={
+                <ActionPanel>
+                  <Action.OpenInBrowser title="Open in browser" url={getCodeDocsUrl(code.code)} />
```

Comment:
```suggestion
                  <Action.OpenInBrowser url={getCodeDocsUrl(code.code)} />
```
Needs to be title cased, but just remove it, it's the default title